core/modules/syncer.js

```diff
@@ -33,6 +33,15 @@ function Syncer(options) {
 	var self = this;
 	this.wiki = options.wiki;
 	this.syncadaptor = options.syncadaptor;
+	if (this.syncadaptor && this.syncadaptor.addEventListener) {
+		this.syncadaptor.addEventListener('server-update', function(type, tiddlerFields){
```

Comment:
Coding standards:

* should use double quotes for string constants
* should not have spaces after the commas in parameter lists